### DISCUSSION TITLE
Last update MS 2020

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6199] [General]
Hi to all,

can anybody confirm that the "init request" via iNit page is not working after the last update :-( ...

![grafik](https://user-images.githubusercontent.com/76531893/142587719-d7bb526c-1923-4fce-8b4a-1315b54d5f5d.png)

By the way....the shining letters are great !!!

greets

**Comment by tracernz** [comment]
This should be fixed by updating A32NX to the latest version.

**Reply by CptBayer** [reply]
thanx to New Zealand :-) ...

as well a manuall programing is not possible ... when pressing LSK1R nothing happend ...

![grafik](https://user-images.githubusercontent.com/76531893/142589863-c3ed83e6-b367-48d5-9c33-f18e7a06bab4.png)

greets

**Reply by tracernz** [reply]
I forgot to say... latest *development* or experimental version. We will do a new stable release in the coming days.

**Reply by CptBayer** [reply]
thanx, i fly mostly development ....

greets

**Reply by CptBayer** [reply]
working again :-) !!!

**Reply by CptBayer** [reply]
ups, after the last development update the INTEG lights aren´t working anymore..

![grafik](https://user-images.githubusercontent.com/76531893/142596371-231a3b21-ac38-47df-93c1-f2203e1ab8d8.png)

greets
